.github/workflows/format.yaml

```
@@ -21,11 +21,34 @@ on:
       - uses: ./.github/actions/dump-context
+  detect-ci-container:
+    permissions:
+      contents: read
+    permissions:
+      contents: read
+    uses: ./.github/workflows/_detect-ci-container.yaml
   format:
+    permissions:
+      contents: read
+    timeout-minutes: 60
+    permissions:
```

actionlint (1.7.7)

key "permissions" is duplicated in "format" job. previously defined at line:46,col:5

(syntax-check)